lib/compilers/spirv.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, 2021, Compiler Explorer Authors, Arm Ltd
+// Copyright (c) 2018, 2021, 2024 Compiler Explorer Authors, Arm Ltd
```

Comment:
Thanks for updating this; though typically we leave the copyright alone but not really for any principled reason...